swift/integration-tests/linux-only/autobuilder/unsupported-os/diagnostics.expected

```diff
@@ -1,7 +1,4 @@
 {
-  "helpLinks": [
```

Comment:
Just to make sure: We're not aware of any place that expects a `helpLink` property to always be present, right?

Reply:
That's correct, they're only used for plaintext messages, which cannot have links embedded in the message body. They're also not required even for plaintext messages.